rclone: update page (#12181)
tldr-pages/tldr: pages/common/rclone.md

> Copy, synchronize or move files and directories to and from many cloud services.
> More information: <https://rclone.org>.

- Launch an interactive menu to setup rclone:

`rclone config`

- List contents of a directory on an rclone remote:

`rclone lsf {{remote_name}}:{{path/to/directory}}`

> - Copy file or directory from the local machine to remote destination:

Reviewer: ```suggestion
- Copy file or directory from the local machine to the remote destination:
```

> - Copy file or directory from the local machine to the remote destination:

Reviewer 2: ```suggestion
- Copy a file or directory from the local machine to the remote destination:
```

`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/directory}}`

> - Copy files changed within the past 24 hours to a remote to the local machine, asking user to confirm each file:

Reviewer: ```suggestion
- Copy files changed within the past 24 hours to a remote from the local machine, asking the user to confirm each file:
```
Is the copying happening from a remote to the local machine, or from a local machine to the remote?
Feel free to reword my suggestion for the same.

`rclone copy --interactive --max-age 24h {{remote_name}}:{{path/to/directory}} {{path/to/local_directory}} `

> - Mirror file or directory (unlike copy, sync removes files from remote if it does not exist on local):

Reviewer: ```suggestion
- Mirror file or directory (unlike copy, sync it removes files from remote if it does not exist locally):
```

Author: I believe the word "it" would make it ungrammatical in this case (also for consistency "Note:" should also be added)

> - Mirror file or directory (Note: Unlike copy, sync removes files from the remote if it does not exist locally):

Reviewer 2: ```suggestion
- Mirror a specific file or directory (Note: Unlike copy, sync removes files from the remote if it does not exist locally):
```

`rclone sync {{path/to/file_or_directory}} {{remote_name}}:{{path/to/directory}}`

> - Delete remote file or directory (`--dry-run` means test, remove it from the command to actually delete):

Reviewer: ```suggestion
- Delete remote file or directory (Note: `--dry-run` means test, remove it from the command to actually delete):
```

> - Delete remote file or directory (Note: `--dry-run` means test, remove it from the command to actually delete):

Reviewer 2: ```suggestion
- Delete a remote file or directory (Note: `--dry-run` means test, remove it from the command to actually delete):
```
